winterfell/webapp/src/js/formRenderingService.js

```diff
@@ -27,3 +27,29 @@ app.factory('formRenderingService', ['$window', '$http', function ($window, $htt
         }
     }
 }]);
+
+/**
+ * Service to retrieve the formly template (which is just a static JSON file)
+ * for a given form.
+ */
+app.factory('formTemplateService', ['$http', function ($http) {
```

Comment:
it adds a wrapper around $http so all you have to provide is a url and body (if needed)